clang/lib/Format/FormatToken.cpp

```diff
@@ -34,9 +34,15 @@ const char *getTokenTypeName(TokenType Type) {
   return nullptr;
 }
 
+// Sorted common C++ non-keyword types.
+static SmallVector<StringRef> CppNonKeywordTypes = {
+    "byte",   "int16_t",  "int32_t",  "int64_t",  "int8_t",
+    "size_t", "uint16_t", "uint32_t", "uint64_t", "uint8_t",
+};
+
 // FIXME: This is copy&pasted from Sema. Put it in a common place and remove
 // duplication.
```

Comment:
Looks like this comment is no longer correct.

Reply:
This comment is still correct. We need to call the function in the lexer to fix it.

Reply:
This function is no longer simply copy/pasted from `Sema`. It doesn't even do the same thing as the Sema function any more.

Reply:
That was the case before this patch though.

Reply:
Before this patch, the two functions were intended to do the same thing, but have apparently diverged. After this patch they are not intended to do the same thing. The FIXME is wrong, can you fix it please?

Reply:
See [here](https://github.com/llvm/llvm-project/pull/83533#issuecomment-1974198544). After this patch,I will add another parameter and call `Token::isSimpleTypeSpecifier()` (which was moved out of `Sema` in #80101) directly. That's the right time to fix (i.e. remove) the comment.